docs/modules/ROOT/pages/reference/extensions/infinispan.adoc

```diff
@@ -38,3 +38,16 @@ Or add the coordinates to your existing project:
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Camel Quarkus limitations
+
+=== InfinispanRemoteAggregationRepository in native mode
+
+At present the `InfinispanRemoteAggregationRepository` is not supported in native mode.
```

Comment:
I wonder what is missing? Do we have an issue for that?

Reply:
https://github.com/apache/camel-quarkus/issues/3657